module/Database/src/Mapper/Member.php

```diff
@@ -117,6 +117,38 @@ public function findAll(): array
         return $this->getRepository()->findAll();
     }
 
+    /**
+     * Find all non-hidden and non-deleted members.
+     *
+     * @return array<array-key, MemberModel>
+     */
+    public function findNormal(): array
+    {
+        return $this->getRepository()->findBy([
+            'deleted' => false,
+            'hidden' => false,
+        ]);
+    }
```

Comment:
In theory expired members should be hidden, so this is good enough (assuming we do not want to return expired members).

Reply:
I think we should avoid adding expired members to external systems; it may cause an issue where someone detects that they forgot to renew their membership/graduate connection. Do we want to build in an extra check here?

Reply:
I was wrong, we currently do not set `hidden` to `true` when a membership has expired. So to be safe we should add the check here.

Reply:
Done by Rink!